.github/workflows/all_components_docker_publish.yaml

```diff
@@ -0,0 +1,145 @@
+      run: |
+        export TAG=$(cat releasing/version/VERSION)
+        cd components/example-notebook-servers/jupyter-pytorch-full
+        export IMG=docker.io/kubeflownotebookswg/notebook-servers/jupyter-pytorch-full
+        docker build -t ${IMG}:${TAG} -f cpu.Dockerfile . 
+        docker push ${IMG}:${TAG}
+
+    - name: Run jupyter-pytorch-cuda-full Notebook Server build and push
+      run: |
+        export TAG=$(cat releasing/version/VERSION)
+        cd components/example-notebook-servers/jupyter-pytorch-full
+        export IMG=docker.io/kubeflownotebookswg/notebook-servers/jupyter-pytorch-cuda-full
+        docker build -t ${IMG}:${TAG} -f cuda.Dockerfile . 
+        docker push ${IMG}:${TAG}
+
+    - name: Run jupyter-tensorflow-full Notebook Server build and push
+      run: |
+        export TAG=$(cat releasing/version/VERSION)
+        cd components/example-notebook-servers/jupyter-tensorflow-full
+        export IMG=docker.io/kubeflownotebookswg/notebook-servers/jupyter-tensorflow-full
+        docker build -t ${IMG}:${TAG} -f cpu.Dockerfile . 
+        docker push ${IMG}:${TAG}
+        
+    - name: Run jupyter-tensorflow-cuda-full Notebook Server build and push
+      run: |
+        export TAG=$(cat releasing/version/VERSION)
+        cd components/example-notebook-servers/jupyter-tensorflow-full
+        export IMG=docker.io/kubeflownotebookswg/notebook-servers/jupyter-tensorflow-cuda-full
+        docker build -t ${IMG}:${TAG} -f cuda.Dockerfile . 
+        docker push ${IMG}:${TAG}
+
+    - name: Run rstudio-tidyverse Notebook Server build and push
+      run: |
+        export TAG=$(cat releasing/version/VERSION)
+        cd components/example-notebook-servers/rstudio-tidyverse
+        export IMG=docker.io/kubeflownotebookswg/notebook-servers/rstudio-tidyverse
+        docker build -t ${IMG}:${TAG} -f Dockerfile . 
+        docker push ${IMG}:${TAG}
+
+    - name: Run PodDefaults build and push
+      run: |
+        export TAG=$(cat releasing/version/VERSION)
+        cd components/admission-webhook
+        export IMG=docker.io/kubeflownotebookswg/poddefaults-webhook
+        make docker-build && make docker-push
+
+    - name: Run Profile Controller build and push
+      run: |
+        export TAG=$(cat releasing/version/VERSION)
+        cd components/profile-controller
+        export IMG=docker.io/kubeflownotebookswg/profile-controller
+        make build
+        make push
+
+    - name: Run Tensorboard Controller build and push
+      run: |
+        export TAG=$(cat releasing/version/VERSION)
+        cd components/tensorboard-controller
+        export IMG=docker.io/kubeflownotebookswg/tensorboard-controller:${TAG}
+        make docker-build
+        make docker-push
+
+    - name: Run TWA build and push
+      run: |
+        export GIT_TAG=$(cat releasing/version/VERSION)
+        cd components/volumes-web-app
+        export IMG=docker.io/kubeflownotebookswg/volumes-web-app
```

Comment:
This one and the above need to be tensorboards, also the path is wrong

Reply:
Fixed that.